app/components/publications/publications-table-row.js

```diff
@@ -25,6 +29,52 @@ export default class PublicationsPublicationsTableRowComponent extends Component
     });
   }
 
+  @task
+  *loadData() {
+    // using queryOne instead of findRecord: findRecord causes included records to be fetched again
+    const publicationFlow = yield this.store.queryOne('publication-flow', {
```

Comment:
Mijn voorstel is om loadDecisions in dezelfde query te verwerken om extra requests te vermijden. Wanneer de conversie heeft gelopen gaat de publicaties-pagina heel traag door erg veel requests.

Reply:
Mag je proberen van mij. Bij toevoegen van `include` in het achterhoofd houden dat dat ervoor kan zorgen dat een cache-entry voor de request sneller geinvalideerd zal worden. De cache clearing is zeer naief/agressief: elke wijziging aan een `decision` zal alle requests met een `include=decision` clearen. Er wordt door de cache dus niet gecheckt of de gewijzigde `decision` diegene is die included is.

Nu zie ik dat in de huidige request van `publication-flow` al andere resources geinclude zijn waarvan de kans veel groter is dat die de cache-entry zullen invalideren dan de extra include van `decision`. Voeg dus maar toe.